### PR TITLE
Walkthrough fixes for EKS, gRPC and HTTP2

### PR DESCRIPTION
**Pull request by lavignes**
* Fix link in EKS base setup to cloudmap walkthrough for doing verification
* Update the verification steps in EKS base to reflect this change
* Remove redundant Mesh resources in gRPC and HTTP2
* Fix bad curl command in http2 walkthrough verification steps

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/eks/base.md [hunk 1]
```diff
@@ -45,29 +50,23 @@ In order to automatically inject App Mesh components and proxies on pod creation
...
-```
+```sh
 helm repo add eks https://aws.github.io/eks-charts
 kubectl create ns appmesh-system
 kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-controller/crds/crds.yaml
 helm upgrade -i appmesh-controller eks/appmesh-controller --namespace appmesh-system
 helm upgrade -i appmesh-inject eks/appmesh-inject --namespace appmesh-system --set mesh.create=true --set mesh.name=color-mesh
 
-Opitionally add tracing
+# Opitionally add tracing
```
Comment (rexnp) [inline walkthroughs/eks/base.md:69]: can you also fix the typo Optionally

File: walkthroughs/eks/base.md [hunk 2]
```diff
@@ -85,135 +84,101 @@ kubectl api-resources --api-group=appmesh.k8s.aws
...
 
 Now, validate the mesh creation using the `aws` CLI:
 
 ```sh
-aws appmesh list-meshes --region us-east-2
+aws appmesh list-meshes
 # {
 #     "meshes": [
 #         {
 #             "meshName": "color-mesh",
```
Comment (rexnp) [inline walkthroughs/eks/base.md:123]: meshName needs to be updated. Similarly on L133, L138, L191

Comment (lavignes) [reply walkthroughs/eks/base.md]: fixed

File: walkthroughs/eks/base.md (hunk 1 above)
Comment (lavignes) [reply walkthroughs/eks/base.md]: done

**Pull request merged**